### ISSUE TITLE
Just want to use on My Winform.

### ISSUE THREAD

**Issue by smallprogrammers** [gmamaladze/globalmousekeyhook#117]
Hello, 
Is it possible Keyboard globally hooked keys will just worked on my Winform Application. Not on other windows. 

Easily to say - If i hooked for A. Then that "A" will not work on any other window. Possible ? 

Thanks in advanced.

**Comment by CrazyTim** [comment]
Instead of using global, use `Hook.AppEvents()`

**Comment by smallprogrammers** [comment]
@CrazyTim  Thanks for your reply. Bro can you tell me one more thing ? I want to hook global. But only my application will capture it. User can't type on notepad or anywhere. They will type But it will only appear in my application. None where else. Possible?

**Comment by smallprogrammers** [comment]
@CrazyTim Thanks for your reply. Bro can you tell me one more thing ? I want to hook global. But only my application will capture it. If user type any charter on Notepad,Word or anywhere that will be not visible on there. Only it will visible in my app.I means it will only appear in my application. None where else. Possible? Suppose someone type APPLE in notepad. But it only show in my app. Not in notepad while hook by my app.

**Comment by zaafar** [comment]
@smallprogrammers give the following lib a try. :)

https://github.com/zaafar/ClickableTransparentOverlay

**Comment by jonjonsson** [comment]
> @CrazyTim I want to hook global. But only my application will capture it. 

Yes you can do that by putting e.handled = true in your keyboard handlers.
